zebrad/src/commands/start/sync.rs

```diff
@@ -88,50 +89,60 @@ where
             self.pending_blocks = Box::pin(FuturesUnordered::new());
 
             tracing::info!("starting sync, obtaining new tips");
-            self.obtain_tips().await?;
+            if self.obtain_tips().await.is_err() {
+                tracing::warn!("failed to obtain tips, waiting to restart sync");
+                delay_for(Duration::from_secs(15)).await;
```

Comment:
We do this delay in two different places now, should we turn it into a constant?

Reply:
Fixed in `67b415cb9bea207d418142f57fce21aa937b572d`.